addons/project_todo/static/tests/todo_conversion_form_view.test.js

```diff
@@ -0,0 +1,43 @@
+import { test, expect } from "@odoo/hoot";
+import { queryAll } from "@odoo/hoot-dom";
+import { animationFrame } from "@odoo/hoot-mock";
+import { mountView, contains, onRpc } from "@web/../tests/web_test_helpers";
+import { defineTodoModels } from "./todo_mock_models";
+import { click } from "@mail/../tests/mail_test_helpers";
+
+
+defineTodoModels();
+
+test("Check that todo_conversion_form view focuses on the focus on the first element", async () => {
+    expect.assertions(2);
+    onRpc("/web/action/load", () => {
+        return {
+            type: "ir.actions.act_window",
+            name: "Convert to Task",
+            res_model: "project.task",
+            view_mode: "form",
+            target: "new",
+            views: [[2, 'form']],
+        };
+    });
+
+    await mountView({
+        resModel: "project.task",
+        resId: 1,
+        type: "form",
+        viewId: false,
+        arch: `
+            <form js_class="todo_form">
+            <field name="name"></field>
+            </form>
+        `,
+        actionMenus: {},
+    });
```

Comment:
```suggestion
    await mountView({
        resModel: "project.task",
        resId: 1,
        type: "form",
        arch: `
            <form js_class="todo_form">
                <field name="name"/>
            </form>
        `,
        actionMenus: {},
    });
```

Is it mandatory to pass viewId as false?